notify.py

```diff
@@ -64,9 +62,14 @@ def __init__(self, url, username, password, room, version):
         self._session.headers.update({'Accept': 'application/json'})
         
         """ Get Token/ID for Room """
-        prefix = "/ocs/v2.php/apps/spreed/api/v1"
-        if self.version >= 12:
-          prefix = "/ocs/v2.php/apps/spreed/api/v4"
+        prefix = "/ocs/v2.php/apps/spreed/api/"
+        if self.session.get(self.url+prefix+"v1/room").status_code == 200:
+            self.api_version = 'v1'
+        if self.session.get(self.url+prefix+"v4/room").status_code == 200:
```

Comment:
the same

Reply:
request_rooms = self._session.get(self.url+prefix+"v4/room")
        if request_rooms.status_code != 200:
            request_rooms = self._session.get(self.url+prefix+"v1/room")